content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,60 @@ tags:
+<br>
+Note that since strafing on ground and in the air causes the player to deviate from their intended destination (assuming it is straight in front of them), prestrafing is usually done only before jumps rather than constantly, and long jumps typically incoroporate multiple left and right airstrafes into a jump which counteract eachother and keep the player close to a straight line.<br><br>
+
+Sometimes you may not be able to take a direct line due to an obstruction. The 2 most common forms of this is rounding a corner (often without being able to see the next platform) and 2 platforms separated by a thin wall.
+In this case, air strafing is not used to gain speed but as a form of air control. The principle is the same but typically smooth turns at the cost of speed are needed (don't take out all your speed or you'll fall straight down!).<br>
+
+### Bhop
+
+Bhopping has reduced potential compared to the Bhop game mode. Bhopping has its own speedcap: **(?) Momkz** &emsp;**380 KZT** &emsp; **300 1.6 Climb**.<br>
+This cap is seperate from the **maxvelocity** cap (how fast the engine lets you go before not bothering to give anymore speed 3500 by default).<br>  
+Anytime you jump the Bhop speed cap will be used meaning even if the player gained speed purely through falling or surfing, as soon as they try to jump off the ground with this speed, the cap will proc.
+If the player reaches this cap and tries to jump again, thier speed will be reset to:<br>
+**? (MomClimb)** &emsp; **? (KZT)** &emsp; **240 (1.6 Climb)**.<br>
+<br> 
+However this Bhop cap still allows the player to gain more speed than a pre-strafe hence, bhopping has the potential to go further than a longjump. Useful to remember when dealing with diffcult longjumps or, on harder maps, a required bhop jump.
+Bhop jumps can also be required by the mapper by using "bhop blocks" that teleport you back if you try standing on them, same as the Bhop gamemode.
+1.6 Climb has a jump-landing penalty, the impact of which can be severely limited with bhops.
+
+### Surf
+
+Surf/Slide is very similar (identical?) to its game mode counterpart with one exception: 1.6 Climb which uses 10aa instead of 100aa.
+This means players can't as easily flick onto a ramp ("board") and genearate speed, or flick off a ramp and generate height. surfs are usually brief and focus on keeping on the board rather than gaining enough speed to fly off, making it more closer to **"sliding"**.
+Maps made with MomClimb and KZT in mind can have more tradtional surf sections.
+Surf has the additional quirk of allowing the user to slide up gentle slopes with just 'W' while looking at the slope
```

Comment:
> Surf has the additional quirk of allowing the user to slide up gentle slopes with just 'W' while looking at the slope

this line is still the same as before, sliding up ramps with w isn't unique to KZ, it's a thing in every movement mode.